Makefile-package.mk

```diff
@@ -0,0 +1,51 @@
+PACKAGE_TYPES     ?= deb rpm
+PROJECT_NAME       = nri-$(INTEGRATION)
+BINS_DIR           = $(TARGET_DIR)/bin/linux_amd64
+SOURCE_DIR         = $(TARGET_DIR)/source
+PACKAGES_DIR       = $(TARGET_DIR)/packages
+VERSION           ?= 0.0.0
+RELEASE           ?= dev
+LICENSE            = "https://newrelic.com/terms (also see LICENSE.txt installed with this package)"
+VENDOR             = "New Relic, Inc."
+PACKAGER           = "New Relic Infrastructure Team <infrastructure-eng@newrelic.com>"
+PACKAGE_URL        = "https://www.newrelic.com/infrastructure"
+SUMMARY            = "New Relic Infrastructure Integrations"
+DESCRIPTION        = "New Relic Infrastructure Integrations extend the core New Relic\nInfrastructure agent's capabilities to allow you to collect metric and\nlive state data from your infrastructure Apache components."
```

Comment:
I suppose SUMMARY should be something like "New Relic Infrastructure Apache integration" or similar... and the description to detail more the Apache integration.